packages/sanity/src/structure/documentActions/DeleteAction.tsx

```diff
@@ -104,5 +103,8 @@ export const DeleteAction: DocumentActionComponent = ({id, type, draft, onComple
   ])
 }
 
-DeleteAction.action = 'delete'
-DeleteAction.displayName = 'DeleteAction'
+useDeleteAction.action = 'delete'
+useDeleteAction.displayName = 'DeleteAction'
+
+/** @internal */
+export const DeleteAction = useDeleteAction
```

Comment:
curious: are we keeping this to preserve the name exported from the package? if so, could/should we rename it internally (including the file) and just keep the old export intact?